### PR TITLE
FirstScreen

### PR DESCRIPTION
**Pull request by amaurycb**
Navigation table pending to review

### CODE REVIEW

File: app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt [hunk 1]
```diff
@@ -33,8 +34,8 @@ class UserListViewModel {
             ) {
                 if (!response.isSuccessful) throw Throwable("error1")
 
-                val result: UserProfile = response.body()?.results?.firstOrNull() ?: UserProfile()
-                lista.value = result
+                val result: List<UserProfile>? = response.body()?.results
```
Comment (williamgdev) [inline app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt]: You do not want to change anything in this class. It is not been used on the app. You can remove it if you are not planning to use it.

File: app/src/main/java/com/example/appempty/SecondFragment.kt [hunk 2]
```diff
@@ -61,7 +62,14 @@ class SecondFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         fullname.setOnClickListener {
-            findNavController().navigate(R.id.action_SecondFragment_to_listaFragment)
+
+            if (resources.getBoolean(R.bool.isTablet)) {
```
Comment (williamgdev) [inline app/src/main/java/com/example/appempty/SecondFragment.kt]: I think you did not test your code. This logic is in the wrong place, there should not be any functionality on this screen when tablet or landscape mode. You might want to run the app and check the ListaFragment button

**Review by williamgdev — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/appempty/SecondFragment.kt (hunk 2 above)
Comment (williamgdev) [reply app/src/main/java/com/example/appempty/SecondFragment.kt]: The fragment list is containing the list of user. and the task description says
_**The User should be able to click in other user and the User Details needs to update dynamically.**_

File: app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt (hunk 1 above)
Comment (amaurycb) [reply app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt]: Esta clase no la estoy "usando" por qué no me funciona. Ajusté el código de ListaFragment.kt   para usar UserViewModel.kt en mientras tenga la clase UserListViewModel disfuncional. Eso era en lo que trabajaba ayer.

Comment (williamgdev) [reply app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt]: Lo que quise decir s que no tiens que modificar nada aqui. porqu eno se esta usando... asi que elimina esos cambios.

Comment (amaurycb) [reply app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt]: Deleted Class

File: app/src/main/java/com/example/appempty/SecondFragment.kt (hunk 2 above)
Comment (amaurycb) [reply app/src/main/java/com/example/appempty/SecondFragment.kt]: Update de secondFragment and put the logic in the ListaFragment

**Review by williamgdev — APPROVED** [review]

**Pull request merged**